rust/query/typeql_update.rs

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+use crate::common::token::Command::Insert;
+use crate::{write_joined, ErrorMessage, Insertable, Query, ThingVariable, TypeQLDelete};
+use std::fmt;
+
+#[derive(Debug, Eq, PartialEq)]
+pub struct TypeQLUpdate {
+    pub delete_query: Option<TypeQLDelete>,
+    pub variables: Vec<ThingVariable>,
```

Comment:
Java stores the variables in two lists: the delete variables and the insert variables. Why do we store just a single list in Rust?